core-ui/src/components/Extensibility/ExtensibilityDetails.js

```diff
@@ -43,12 +44,16 @@ export const ExtensibilityDetailsCore = ({ resMetaData }) => {
       windowTitle={t('name', {
         defaultValue: resMetaData.resource?.kind,
       })}
-      customColumns={header.map(def => ({
-        header: widgetT(def),
-        value: resource => (
-          <Widget value={resource} structure={def} schema={schema} />
-        ),
-      }))}
+      customColumns={
+        isArray(header)
+          ? header.map(def => ({
+              header: widgetT(def),
+              value: resource => (
+                <Widget value={resource} structure={def} schema={schema} />
+              ),
+            }))
+          : []
+      }
       customComponents={[
         resource => (
           <Widget value={resource} structure={body} schema={schema} />
```

Comment:
Shouldn't we check if `body` is an array as well? If it is an object we display it on the page.
<img width="560" alt="Screen Shot 2022-06-10 at 12 14 22" src="https://user-images.githubusercontent.com/20790348/173044849-533359af-d248-40e5-8ef6-fcb552af8444.png">
<img width="572" alt="Screen Shot 2022-06-10 at 12 15 15" src="https://user-images.githubusercontent.com/20790348/173044854-879e7558-bc37-4624-a29d-3409643b6f8c.png">